src/main/kotlin/com/exactpro/th2/processor/strategy/AbstractCrawlerStrategy.kt

```diff
@@ -109,6 +112,20 @@ abstract class AbstractCrawlerStrategy(context: Context): AbstractStrategy(conte
 
     internal abstract fun createCradleMessageGroupCrawler(context: Context, processor: IProcessor): Crawler<*>
 
+    private fun checkProcessingDelayAndWait() {
```

Comment:
I can implement this test only if I move the waiting functionality to the `Context` class. If you are okay with that then sure, I can add tests for this case

Reply:
Could we implement integration test, where crawler starts from current interval and sends the first request after processing delay timeout?

Reply:
This will only verify that the crawler works. But this will not cover the fact we waited for something indeed...
I can add event generation to generate the event when the strategy is waiting for some time to pass

Reply:
Ok, let's do it